util/etcd/election_test.go

```diff
@@ -54,3 +60,89 @@ func TestMasterElectionThroughCommonClient(t *testing.T) {
 		t.Fatalf("Close(20): %v", err)
 	}
 }
+
+func everyoneAgreesOnMaster(ctx context.Context, t *testing.T, want string, elections []util.MasterElection) {
+	t.Helper()
+	for _, e := range elections {
+		for {
+			log.Print("Getting master")
+			m, err := e.GetCurrentMaster(ctx)
+			if err == concurrency.ErrElectionNoLeader {
+				t.Error("No leader...")
+				time.Sleep(time.Second)
+				continue
+			} else if err != nil {
+				t.Fatalf("Failed to GetCurrentMaster: %v", err)
+			}
+			if m != want {
+				log.Printf("got %v want %v", m, want)
```

Comment:
`t.Logf` preferred, but maybe just drop this log altogether?

Reply:
done